### DISCUSSION TITLE
绳网取代jm计划

### DISCUSSION THREAD

**Discussion by tingsha1935** [share121/inter-knot#2258] [其他]
![e3030d6f2cf7a5366b86c0cda7d9015](https://github.com/user-attachments/assets/fa636d90-05ad-4c96-b5eb-331ab71ccf19)
v我50菲林，聆听我的宏图霸业

**Comment by ShineStarAurora** [comment]
我不要

**Comment by Hejin1212** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/11/16/810dd11a4d16759f16ca078596aab6b1_422007669816866994.png" alt="哲-让我想想" width="75" >不要

**Comment by Orangered0801** [comment]
汗流浃背了

**Comment by Genshin-Neuvillette** [comment]
不行，色色是色色，玩是玩<img src="https://img-static.mihoyo.com/communityweb/upload/5591a162112cc3c3a3c12c1e5a406020.png" alt="辛焱-气死了" width="75" >

**Comment by penglaisi** [comment]
我是魔神芙卡洛斯，V我50原石，待我归来，封你做最高审判官！

**Comment by 666waht** [comment]
我都有JM了，为什么要用绳网呢？还要挂梯子

**Comment by anything12121** [comment]
逆天了

**Comment by QiyueningXue** [comment]
布什 戈门

**Comment by VacuolePaoo** [comment]
此帖涉嫌诈骗

**Comment by lllliufeng** [comment]
啊？

**Comment by YuTinMIn** [comment]
小心被人举报了，但私聊给我可以（划掉）
